src/Explorer/Panes/AddCollectionPanel.tsx

```diff
@@ -1426,9 +1426,8 @@ export class AddCollectionPanel extends React.Component<AddCollectionPanelProps,
       TelemetryProcessor.traceSuccess(Action.CreateCollection, telemetryData, startKey);
       useSidePanel.getState().closeSidePanel();
       // open NPS Survey Dialog once the collection is created
-      if (userContext.features.enableNPSSurvey) {
         this.props.explorer.openNPSSurveyDialog();
```

Comment:
nit - it doesn't look like the indent has been removed to align this with the rest of the code ?